.bowerrc

```diff
@@ -0,0 +1,3 @@
+{
+    "directory": "static/libs/bower_components"
```

Comment:
One more dependency. Great :(
_-- Just my opinion. It is neither recommendation nor changes request._